packages/ui/src/components/ProfileItem.tsx

```diff
@@ -0,0 +1,77 @@
+import { icons } from "lucide-react";
+import { ComponentProps } from "react";
+
+import { cn } from "@ui/lib/utils";
+
+type ProfileItemProps = {
+  children: React.ReactNode;
+} & ComponentProps<"div">;
+
+function ProfileItem({ className, children, ...props }: ProfileItemProps) {
+  return (
+    <div
+      className={cn(
+        `bg-background-primary relative flex min-h-[2.813rem] min-w-[22.375rem] items-center justify-between`,
+        className,
+      )}
+      {...props}
+    >
+      {children}
+    </div>
+  );
+}
```

Comment:
아래 리뷰들에서 컴포넌트들의 usecase가 명확한 것에 비해 컴포넌트 설계가 너무 많은 범용성을 지원한다고 피드백드렸습니다. 
제 생각에는 이렇게 usecase가 명확하고 컴포넌트 엘리먼트 및 스타일도 동일한 경우, ProfileItem props로 `type`이나 `variant`를 받아 값에 따라 ProfileItemHeader와 ProfileItemIcon을 알아서 렌더링해주는 게 어떨까 합니다.
```suggestion
const ProfileItemVariants = {
   birthday: {
      icon: BirthdayIcon,
      content: '생년월일'
    },
    phoneNumber: {
       icon: PhoneIcon,
       content: '전화번호',
    }
}
function ProfileItem({ variant, className, children, ...props }: ProfileItemProps) {
  return (
    <div
      className={cn(
        `bg-background-primary relative flex min-h-[2.813rem] min-w-[22.375rem] items-center justify-between`,
        className,
      )}
      {...props}
    >
      <ProfileItemIcon iconName={ProfileItemVariants[variant].icon} />
      <ProfileItemHeader content={ProfileItemVariants[variant].content} />
      {children}
    </div>
  );
}
```

Reply:
이렇게 하면 ProfileItem을 사용하는 개발자는 ProfileItemIcon과 ProfileItemHeader의 렌더링에 신경 쓰지 않아도 되며, ProfileItemContent의 로직만 주입하면 돼서 사용 시 더 명확할 것 같다는 개인적인 생각입니다

Reply:
유스케이스가 명확한 상황에선 용재님의 의견대로 타입을 좁혀주어 props를 제공하는 게 개발자 입장에서 렌더링을 신경쓰지 않아도 되어 더욱 편리한 것 같네요. 설계 좋은 것 같습니다.